README.md

```diff
@@ -0,0 +1,69 @@
+## Case Classy
+[![Build Status](https://api.travis-ci.org/47deg/case-classy.png?branch=master)](https://travis-ci.org/47deg/case-classy)
+[![codecov.io](http://codecov.io/github/47deg/case-classy/coverage.svg?branch=master)](http://codecov.io/github/47deg/case-classy?branch=master)
+
+### Introduction
+
+Case classy is a tiny framework to make it easy to decode untyped
```

Comment:
I think given its size library would be more appropriate than framework. Also if we are still under the zerop deps goal this may be a good place to mention that.